internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -1114,6 +1114,7 @@ func resourceKubernetesCluster() *pluginsdk.Resource {
 				ValidateFunc: validation.StringInSlice([]string{
 					string(managedclusters.ManagedClusterSKUTierFree),
 					string(managedclusters.ManagedClusterSKUTierPaid),
+					string(managedclusters.ManagedClusterSKUTierStandard),
```

Comment:
this'll also want documenting - or was this supposed to be part of another PR?

Reply:
Nope, I forgot. Thanks!